code/baseStation/base_parser3.py

```diff
@@ -102,22 +113,21 @@ def thermal_get(self):
             self.thermal_counter += 1
             print(self.thermal_counter)
             print("T", thermal_line)
-            parsed_line = ["%.1f" % (ord(el)/10.0) for el in thermal_line]  # Decompression
+            parsed_line = ["{0:.1f}".format(el/10.0) for el in thermal_line]  # Decompression
             self.thermal.write(" ".join(parsed_line))
             self.thermal.write('\n')
 
         self.thermal_lastline = thermal_line
         if self.thermal_counter == THERMAL_HEIGHT:
             self.thermal.close()
 
-
     def system(self):
         """
         Prints the received system status.
         Triggers upon receiving an 'SS' header.
         """
-        print("System: ", "".join(self.line[2:]).strip('-'))
-
+        msg = "".join(bytes(self.line[2:]).decode("utf-8")).strip('-')
+        print("SYSTEM: ", msg)
```

Comment:
Why is printing still a thing while we're using `logging` already?

Reply:
In base station it's good to see live what are we receiving, isn't it?